src/docs/index.adoc

```diff
@@ -15,12 +15,12 @@
 ** link:PSA/PSAUserGuide.html[*Service Automation*] (PaaS On-Demand, Service deployment and management)
 * link:admin/ProActiveAdminGuide.html[*PWS Admin Guide*] (Installation, Infrastructure & Nodes setup, Agents,…​)
 
-=== Machine Learning Open Studio (MLOS)
+=== Machine Learning Open Studio (PML)
```

Comment:
Could you please update _Machine Learning Open Studio (PML)_ to _ProActive Machine Learning (PML)_